### ISSUE TITLE
The "ray start" command is unusable without an internet connection.

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#351]
To reproduce, turn off your internet, and run

```bash
./scripts/start_ray.sh --head
```

It will fail with

```python
Traceback (most recent call last):
  File "/Users/rkn/Workspace/ray/scripts/start_ray.py", line 51, in <module>
    node_ip_address = services.get_node_ip_address()
  File "/Users/rkn/Workspace/ray/python/ray/services.py", line 164, in get_node_ip_address
    s.connect((ip_address, int(port)))
OSError: [Errno 51] Network is unreachable
```

**Comment by robertnishihara** [comment]
The relevant command now is `ray start --head`.

**Comment by robertnishihara** [comment]
Fixed by #1556.
